src/darker/black_diff.py

```diff
@@ -131,6 +131,9 @@ def apply_black_excludes(
     :return: Absolute paths of files which should be reformatted using Black
 
     """
+    sig = inspect.signature(gen_python_files)
+    # those two exist and are required in black>=21.8b0
+    kwa = dict(verbose=False, quiet=False) if "verbose" in sig.parameters else {}
```

Comment:
Just a stylistic thing, I'd use the more commonly seen name `kwargs` for the extra keyword arguments.
```suggestion
    kwargs = dict(verbose=False, quiet=False) if "verbose" in sig.parameters else {}
```